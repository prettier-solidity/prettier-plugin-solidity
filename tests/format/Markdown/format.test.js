run_spec(import.meta, ['markdown']);
