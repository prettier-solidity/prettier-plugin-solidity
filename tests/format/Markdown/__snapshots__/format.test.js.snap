// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown.md format 1`] = `
====================================options=====================================
parsers: ["markdown"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
# Embedded Solidity Code

## Original

<!-- prettier-ignore -->
\`\`\`solidity
import    "./Foo.sol";
\`\`\`

## Formatted
\`\`\`solidity
import    "./Foo.sol";
\`\`\`

=====================================output=====================================
# Embedded Solidity Code

## Original

<!-- prettier-ignore -->
\`\`\`solidity
import    "./Foo.sol";
\`\`\`

## Formatted

\`\`\`solidity
import "./Foo.sol";
\`\`\`

================================================================================
`;
